db/migrate/20160726032542_add_counter_to_game.rb

```diff
@@ -0,0 +1,5 @@
+class AddCounterToGame < ActiveRecord::Migration
+  def change
+    add_column :games, :move_counter, :string
```

Comment:
You probably want this to be `:integer` rather than a string.  Also you probably want to add `, default: 0` or whatever to control the default value for new games.